File: CodeForces/BoyorGirl.java
import java.util.Scanner;

public class Solution {
    public static void main(String[] args) {

        Scanner scan = new Scanner(System.in);

        System.out.println(scan.nextLine().chars().distinct().count() % 2 == 0 ? "CHAT WITH HER!" : "IGNORE HIM!");

    }
}

// https://codeforces.com/problemset/problem/236/A

File: CodeForces/BoyorGirl.py
name = input()
newList = []

for i in name:
    if i not in newList:
        newList.append(i)

if (len(newList) % 2 == 0):
    print("CHAT WITH HER!")
else:
    print("IGNORE HIM!")


# https://codeforces.com/problemset/problem/236/A
